db/migrate/20221025151600_add_created_by_user_id_to_hosts.rb

```diff
@@ -0,0 +1,5 @@
+class AddCreatedByUserIdToHosts < ActiveRecord::Migration[6.1]
+  def change
+    add_column :hosts, :created_by_user_id, :integer
```

Comment:
I'd prefer if this was a proper foreign key to a user. If the user is deleted, it should be set to NULL because an ID doesn't really say anything. We have the audits table if people really want to know what happened anyway.

Reply:
Yeah, I wasn't sure what to decide regarding this.
Thanks for the comment, I'll add it.